src/core/capture.py

```
@@ -0,0 +1,190 @@
+"""Screen capture functionality."""
+
+from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)